web/src/main/java/org/verwandlung/voj/web/service/ContestService.java

```diff
@@ -167,7 +167,7 @@ public Contest getContest(long contestId) {
 			Map<Long, Map<String, String>> codeSnippet = JSON.parseObject(cc.getCodeSnippet(),
 					new TypeReference<Map<Long, Map<String, String>>>() {});
 
-			if ( codeSnippet.containsKey(problemId) ) {
+			if ( codeSnippet!=null && codeSnippet.containsKey(problemId) ) {
```

Comment:
Please change to `codeSnippet != null`.
Make sure to leave blanks before and after `!=`.